lib/protocol/http/header/cache_control.rb

```diff
@@ -55,11 +59,40 @@ def no_store?
 					self.include?(NO_STORE)
 				end
 				
+				# Indicates that a response must not be used once it is stale.
+				# See https://www.rfc-editor.org/rfc/rfc9111.html#name-must-revalidate
+				def must_revalidate?
+					self.include?(MUST_REVALIDATE)
+				end
+				
+				# Like must-revalidate, but for shared caches only.
+				# See https://www.rfc-editor.org/rfc/rfc9111.html#name-proxy-revalidate
+				def proxy_revalidate?
+					self.include?(PROXY_REVALIDATE)
+				end
+				
+				# The maximum time, in seconds, a response should be considered fresh.
+				# See https://www.rfc-editor.org/rfc/rfc9111.html#name-max-age-2
 				def max_age
 					if value = self.find{|value| value.start_with?(MAX_AGE)}
 						_, age = value.split('=', 2)
 						
-						return Integer(age)
+						if age =~ /^[0-9]+$/
```

Comment:
Please use `\A` / `\z`. Maybe put this into a private method (`parse_integer`?).

Reply:
Done!